src/player/systems/mod.rs

```diff
@@ -20,8 +19,7 @@ use thetawave_interface::player::PlayerComponent;
 use thetawave_interface::run::{RunDefeatType, RunEndEvent, RunOutcomeType};
 use thetawave_interface::spawnable::EffectType;
 
-pub use self::ability::*;
-pub use self::attacks::{fire_weapon_system, scale_fire_rate_system};
+pub use self::abilities::*;
```

Comment:
Let's not use `pub use`. It is helpful in libraries for exposing a public API, but just makes navigation more confusing in applications. Especially in the age of auto import